share/zm-centos.sh

```diff
@@ -8,9 +8,9 @@
 # Should-Stop:
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Short-Description: Start and stop the Zonemaster Web Backend demons
+# Short-Description: Start and stop the Zonemaster RPC API demons
```

Comment:
This file will start both the RPC API daemon and the Test Agent daemon. Then the description should be
"Start and stop the Zonemaster Backend (RPC API daemon and Test Agent daemon)"